src/lexical-elements.rst

```diff
@@ -618,33 +598,50 @@ The :t:`type` of a :t:`simple byte literal` is :c:`u8`.
 .. _fls_fqaffyrjob7v:
 
 Byte String Literals
-^^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~~
 
 .. rubric:: Syntax
 
 .. syntax::
-
    ByteStringLiteral ::=
-       $$b"$$ ByteStringContent* $$"$$
+       RawByteStringLiteral
+       | SimpleByteStringLiteral
 
-   ByteStringContent ::=
+.. rubric:: Legality Rules
+
+:dp:`fls_t63zfv5JdUhj`
+A :t:`byte string literal` is a :t:`literal` that consists of multiple byte characters.
```

Comment:
To make byte string literal symmetric with simple byte string literal, this one should probably consist of multiple :s:`AsciiCharacter`s.